### ISSUE TITLE
It doesn't annonce the new value with JAWS 16

### ISSUE THREAD

**Issue by antoninadert** [schne324/electric-slide#1]
I tried it on JAWS 16 but it doesn't announce the new value when I change the value with up and down arrow.
